specfile/specfile.py

```diff
@@ -611,7 +611,12 @@ def add_changelog_entry(
                 if self.contains_autochangelog(section):
                     continue
                 if evr is None:
-                    evr = "%{?epoch:%{epoch}:}%{version}-%{release}"
+                    epoch_prefix = (
+                        f"{self.expanded_epoch}:" if self.expanded_epoch else ""
+                    )
+                    evr = (
+                        f"{epoch_prefix}{self.expanded_version}-{self.expanded_release}"
+                    )
```

Comment:
```suggestion
                    with self.tags() as tags:
                        try:
                            evr = f"{tags.epoch.expanded_value}:"
                        except AttributeError:
                            evr = ""
                        evr += f"{tags.version.expanded_value}-{tags.release.expanded_value}"
```

Reply:
Yeap, units are failing with this. Shall we revert?